user/pages/03.Howtos/15.repartition-vm-disk/docs.en.md

```diff
@@ -67,7 +134,7 @@ tmpfs           997M     0  997M   0% /sys/fs/cgroup
 tmpfs           200M     0  200M   0% /run/user/1000
 ```
 
-### Manually configure partitions
+### Example 2 : (Manually) configure partitions
```

Comment:
```suggestion
### Example 3 : (Manually) configure partitions
```
I'm all for sequential counting. Apart from that, I see the didactical order inversed. Shouldn't we start with the simplest approach and go towards the more sophisticated (but nonetheless recommended) ones?